app/assets/stylesheets/new_admin.css

```diff
@@ -0,0 +1,4 @@
+img {
```

Comment:
When writing css, you need to scope the css so that it only applies to a specific thing.